spec/features/users/user_can_comment_on_question_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe 'when a user visits a question show page' do
```

Comment:
For testing, the before blocks are really helpful for cleaning up your test setup a little bit.